write.go

```diff
@@ -70,6 +87,19 @@ func Write(out io.Writer, ds Dataset, opts ...WriteOption) error {
 	return nil
 }
 
+// WriteElement writes a single DICOM element to a Writer
+func (w *Writer) WriteElement(e *Element) error {
+	return writeElement(w.writer, e, *w.optSet)
+}
+
+// Write will write the input DICOM dataset to the provided io.Writer as a complete DICOM (including any header
+// information if available).
+func Write(out io.Writer, ds Dataset, opts ...WriteOption) error {
```

Comment:
hm, so now that we have both `dicom.Write` and `w.WriteDataset`, those are essentially two of the same ways of writing out the full dataset. I'd like to be opinionated, and if the user wants to just dump a DICOM dataset to disk there should be one clear way to do that (ideally, by just calling `dicom.Write`). I need to think on that a little. Maybe the solution is to make `w.writeDataset` unexported. If the user is using the `Writer` struct, that's the "manual approach" where they have full control. 

This also matches the `Parse` and `Parser` pattern...

Need to think on it a little, let me know what you think